pilot/pkg/config/kube/crd/conversion_test.go

```diff
@@ -38,11 +39,17 @@ func TestConvert(t *testing.T) {
 			Annotations:      map[string]string{"annotation": "value"},
 		},
 		Spec: mock.ExampleVirtualService,
+		Status: v1alpha1.IstioStatus{
+			Conditions: []*v1alpha1.IstioCondition{
+				{Type: "Health"},
+			},
+		},
 	}
 
 	obj, err := ConvertConfig(config)
 	if err != nil {
-		t.Errorf("ConvertConfig() => unexpected error %v", err)
+		t.Errorf("Conv"+
+			"ertConfig() => unexpected error %v", err)
```

Comment:
Any reason not to directly write `ConvertConfig()`?

Reply:
that was a mistake lol

Reply:
Can you update this?